docs/.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Maybe remove these comments since that's no valid JSON?

Reply:
Did not want to commit that...